Tests/Features/Context/VictoireContext.php

```diff
@@ -132,6 +132,17 @@ public function iLoginAsVisitor()
         $this->minkContext->setMinkParameter('base_url', $url);
     }
 
+    /**
+     * @Given I login as Victoire default user
+     */
+    public function iLoginAsVictoireDefaultUser()
+    {
+        $this->getSession()->getDriver()->stop();
+        $baseUrl = $this->minkContext->getMinkParameter('base_url');
+        $url = str_replace('anakin@victoire.io:test', 'luke@victoire.io:test', $baseUrl);
```

Comment:
I'm not sure about this... I would rather to pass the user email to the function parameters
```diff
    /**
-     * @Given I login as Victoire default user
+     * @Given I change default user
+     * @Given I use :email as default user
     */
-    public function iLoginAsDefaultUser()
+    public function iChangeDefaultLoginUser($email = 'luke@victoire.io')
    {
        $this->getSession()->getDriver()->stop();
        $baseUrl = $this->minkContext->getMinkParameter('base_url');
-       $url = str_replace('anakin@victoire.io:test', 'luke@victoire.io:test', $baseUrl);
+       $url = str_replace('anakin@victoire.io:test', $email.':test', $baseUrl);
```

Reply:
Yes, i'v made some change, now i use a generic function to change default login user